mne/viz/raw.py

```diff
@@ -669,6 +712,7 @@ def plot_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf, proj=False,
     -------
     fig : instance of matplotlib figure
         Figure with frequency spectra of the data channels.
+
```

Comment:
I don't think we need this and it might actually cause the style checkers some anguish

Reply:
Removed.